lib/tool-bar.js

```diff
@@ -29,8 +29,8 @@ exports.config = {
   },
   iconSize: {
     type: 'string',
-    default: '24px',
-    enum: ['12px', '16px', '24px', '32px'],
+    default: '21px',
```

Comment:
I'm personally not sure if changing the default is a good idea (also given my comment above).

Reply:
I don't feel strongly about it. We can return to 24. Whatever you guys think.

Reply:
I'd definitely keep the 24px default. There's no reason for changing it.